.github/workflows/integration-tests-benchmarks.yml

```diff
@@ -4,13 +4,26 @@ on:
     branches:
       - main
   pull_request:
+    paths:
+      - '**/sentry/**'
+      - '**/sentry-android/**'
+      - '**/sentry-android-core/**'
+      - '**/sentry-android-ndk/**'
+      - '**/sentry-android-integration-tests/sentry-uitest-android-benchmark/**'
+      - '**/.github/**'
 
 jobs:
   test:
     name: Benchmarks
     runs-on: ubuntu-latest
 
     steps:
+      - name: Check pr keyword
+        uses: JJ/github-pr-contains-action@releases/v4
```

Comment:
could you pin it to a specific commit hash, so it's reproducible in case of problems?

Reply:
i'm just making some tests, it's not meant to be merged right now
it seems the workflow fails instead of being cancelled, so maybe it's not a viable option (or can we "force" a merge in case a check fails?)

Reply:
Can we do similar but for comments maybe? https://stackoverflow.com/a/59775665

Reply:
i removed the logic to skip based on the pull request description.
We will consider to add a check on the git commit message, in case benchmarks are still taking a lot to run